echopype/tests/convert/test_parsed_to_zarr.py

```diff
@@ -90,6 +90,9 @@ def test_raw2zarr(raw_file, sonar_model, offload_to_zarr, ek60_path):
         # If it goes all the way to here it is most likely successful
         assert os.path.exists(output_save_path)
 
+    if offload_to_zarr:
+        del echodata
```

Comment:
Wondering if it'd be worthwhile to check whether the created temp dir was successfully deleted after this step, since in our CI there shouldn't be the jupyter problems.

Reply:
That sounds like a good idea, I will add that.